### PR TITLE
:sparkles: Overhaul Gitpod Website Homepage

### LINKED ISSUE #3382: homepage sprint (part 5): content updates
You can find the mockup [here](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=8898%3A17416&t=xSdAndqYzGskFOW4-1)

**Tasks**

- [x] update subheadline to "Ship software faster with secure cloud  development environments that just work."
- [x] change headline for trusted by section to "Speeding up +750k developers in teams like"
- [x] update success metrics (content + add a caption) 
Bildschirm­foto 2023-02-06 um 13 36 44

- [x] add "what's a CDE?" section (overhauls the old "Your environment, your tools, your craft" section)
- [x] New IDE Component: all screenshots have been updated. You can find them [here](https://www.figma.com/file/VL0CJpHqcDJVJ4BEkoyRFV/Gitpod-Website-Component-Library?node-id=2056%3A3271&t=y5hfRPbP2Ljoom8K-1) (Screenshots for Rider, Clion and Vim are still WIP)
- [x] New Toggle: default selection: Desktop & VS Code

- [x] update content and order of the feature section (see [here](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=8898%3A17416&t=xSdAndqYzGskFOW4-1))
- [x] add benefit overview 
Bildschirm­foto 2023-02-06 um 13 43 23

Let me know in case you need a mockup for dark theme or mobile @Siddhant-K-code 🙌

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
fixes #3382

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3387"></a>

### CODE REVIEW

**Comment by Siddhant-K-code** [comment]
Feature components are not aligned (it is still - an alternate left/right). I don't think, we should block the PR because of it.

**Comment by MichaelAring** [comment]
The page is coming along great!

I noticed the boxes have different sizes:
Screenshot 2023-02-08 at 13 56 49

Also, it should say "Increase security" not "Increased security"

**Comment by Siddhant-K-code** [comment]
I am also a fan of standard boxes. But [in design](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=8898%3A17416&t=2XnPVRH44WRgDqG1-4), I found the variable size so implemented that


image

**Comment by Siddhant-K-code** [comment]
> I noticed the boxes have different sizes:
> Screenshot 2023-02-08 at 13 56 49
> 
> Also, it should say "Increase security" not "Increased security"
> 

Resolved in [`0e07106`](https://github.com/gitpod-io/website/pull/3387/commits/0e07106ebef6d69e7ad2b4eb32bd39abf4a2ab12)

**Comment by Siddhant-K-code** [comment]
Feedbacks, Sales, Support forms are working fine ✅

**Comment by Siddhant-K-code** [comment]
It is ready to be merged from my side! ( I optimized for all mobile, dark mode) If there is any minor CSS change, we can address this in follow-up PRs. Also, a [suggestion](https://github.com/gitpod-io/website/pull/3387#issuecomment-1422462182) to consider.


// cc: @chrifro

**Review by chrifro — COMMENTED** [review]
Awesome, thanks @Siddhant-K-code ✨

Some comments:

- [ ] the animated quotes should start a bit earlier now that we don't have the contact pop-up anymore
- [ ] the screenshots for IDE selector are not updated. They all got a new design, see [here](https://www.figma.com/file/VL0CJpHqcDJVJ4BEkoyRFV/Gitpod-Website-Component-Library?node-id=2056%3A3271&t=xkT52a2tNBOZytJB-1)
- [ ] the hover and active states of the IDE selector don't align with mockup yet 
active: bg-> white, stroke 3px and grey, no shadow
hover: bg -> white, linear stroke 1px, no shadow

- [ ] reduce spacing to 160px
Bildschirm­foto 2023-02-08 um 14 44 36

- [ ] align icon size. The JetBrains IDE icons are a lot bigger than the other icons. Let's align them to the smaller style. Because of the spacing inside the cards is also not even

Bildschirm­foto 2023-02-08 um 14 46 38

- [ ] the boxes are not aligned in the width
Bildschirm­foto 2023-02-08 um 14 50 27

- [ ] align spacing to 48px (we shouldn't have any spacing for 88px defined)
Bildschirm­foto 2023-02-08 um 14 50 58

- [ ] dark theme: change design to this
Bildschirm­foto 2023-02-08 um 15 00 16

- [ ] mobile layout still needs some improvements. I will create mockups for them

Great first iteration @Siddhant-K-code 👏

**Comment by Siddhant-K-code** [comment]
> mobile layout still needs some improvements. I will create mockups for them

are we gonna block this PR due to this?

**Comment by Siddhant-K-code** [comment]
> the animated quotes should start a bit earlier now that we don't have the contact pop-up anymore

didn't get this 🤔

File: src/lib/contents/home/features.ts [hunk 1]
```diff
@@ -14,10 +14,32 @@ export const multiTrackDevelopmentFeature: Feature = {
...
+    height: "100%",
+  },
+};
+
 export const collaborationFeature: Feature = {
-  title: "Where teams code together",
+  title: "Collaborate with ease",
   paragraph:
-    "Share running workspaces for pair programming, use port forwarding or share a snapshot as a copy of your workspace with teammates.",
+    "Engineers, designers, product managers. With Gitpod, anyone on the team can run code with ease. Yes, your CTO as well 🫶</br></br>Share running workspaces for pair programming, use port forwarding or share a snapshot of your CDE.",
```
[the remark this answers is not in the document]
Comment (Siddhant-K-code) [reply src/lib/contents/home/features.ts:42]: It's there right? (after the end of sentence)

**Comment by chrifro** [comment]
>>mobile layout still needs some improvements. I will create mockups for them

>are we gonna block this PR due to this?

No, it would be great to get all issues above the line (in my comment) done before the All-hands. All other comments including the mobile improvement can be addressed afterward in al follow-up PR. Do you think that's possible?

**Comment by Siddhant-K-code** [comment]
It is due to shadow!

image

**Comment by chrifro** [comment]
> >the animated quotes should start a bit earlier now that we don't have the contact pop-up anymore

>didn't get this 🤔

I moved it to the follow-up section. Currently, we have a delay before this animation starts. We need to reduce the time of the delay
Bildschirm­foto 2023-02-08 um 15 14 51

**Comment by Siddhant-K-code** [comment]
done :) 

image

**Pull request merged**